src/confdb/data/ConfigInfo.java

```diff
@@ -5,238 +5,238 @@
 import java.util.ArrayList;
 import java.util.Collections;
 
-
 /**
- * ConfigInfo
- * ----------
+ * ConfigInfo ----------
```

Comment:
mhm, no, this was supposed to stay on the next line

Reply:
OK. Will revert manually.